packages/patternfly-4/react-core/src/components/Page/PageSection.js

```diff
@@ -19,6 +19,8 @@ const propTypes = {
   variant: PropTypes.oneOf(Object.values(PageSectionVariants)),
   /** Modifies a main page section to have no padding */
   noPadding: PropTypes.bool,
+  /** Modifies a main page section to have filling/no-filling */
+  fillVariant: PropTypes.bool,
```

Comment:
Since this is a boolean, I think a better name might be `isFilled` (That is what we have for other components like Card).  And I might update the description to say 'Enables the page section to fill the available vertical space`